Source/WebKit/UIProcess/Automation/WebDriverBidiProcessor.cpp

```diff
@@ -105,7 +178,40 @@ void WebDriverBidiProcessor::sendBidiMessage(const String& message)
     LOG(Automation, "[s:%s] sendBidiMessage of length %d", session->sessionIdentifier().utf8().data(), message.length());
     LOG(Automation, "%s", message.utf8().data());
 
-    session->sendBidiMessage(message);
+    auto msgValue = JSON::Object::parseJSON(message);
```

Comment:
This code is performing the correct operations to unpack the Inspector protocol message.. but I believe it should be performed in WebDriverService or WebDriverServer where the `Automation.bidiMessageSent` event is handled. Rather than on the UIProcess side.

If we construct the object here, we won't have the original id supplied by the client when the command was first received, right?